### ISSUE TITLE
A few SSR issues with utils fns

### ISSUE THREAD

**Issue by gtallest** [jossmac/react-scrolllock#30]
Hi Joss, first of all, thanks for your hard work on this lib!

I've run into a few issues with SSR when using the lib, which appear to be coming from the utility functions:

![getpadding](https://user-images.githubusercontent.com/6742920/41737655-c1082892-7544-11e8-8be1-cb9b81cd8614.jpg)
![getdocumentheight](https://user-images.githubusercontent.com/6742920/41737663-c3d3bea6-7544-11e8-95fc-bd939bd59022.jpg)

I was able to alleviate the issues locally by checking for `typeof window` wherever a util fn was checking for `document` or `window` like such:

```js
function getPadding() {
  if (typeof window === 'undefined' || !window || !document) return 0;
  ...
}

function getDocumentHeight() {
  if (typeof window !== 'undefined' && document && document.body) {
  ...
}

// these fns also check for window but weren't giving me any issues at the time
function isTouchDevice() {...}
function getWindowHeight() {...}
```

I haven't run into any problems with the above, but I realize it may not be all encompassing or I may be missing something. I'm happy to help with anything, so if you need me to make any changes or want me to make a PR, just let me know!

Thanks,
-Marc

**Comment by pauloportella** [comment]
Hi all, 
Thanks for the package, I've run into the same issues when trying to create a production build. Marc fix worked for me. 
Thanks

**Comment by jossmac** [comment]
@gtallest a PR with these fixes would be amazing! Thank you for offering 😍

**Comment by gtallest** [comment]
@jossmac just opened!

https://github.com/jossmac/react-scrolllock/pull/31

**Comment by jossmac** [comment]
Resolved by #31

Thank you @gtallest -- sorry I missed this
